File: MovieBackend/MovieBackend/Models/ListItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MovieBackend.Models
{
    public class ListItem
    {
        [System.ComponentModel.DataAnnotations.Key]

        public string Name { get; set; }
        public string DateAdded { get; set; }
        public int Movie_ID { get; set; }
        public int User_ID { get; set; }
    }
}

File: MovieFrontEnd/src/app/movies/list_item/listitem.ts
import { MovieModel } from './moviemodel';
import { SearchdMovie } from './searchedmovie';

export interface ListItem {
    "name": string,
    "dateAdded": string,
    "movie_id": number,
    "user_id": number,
    "movielist": Array<MovieModel>
    "searchlist": Array<SearchdMovie>
}